### ISSUE TITLE
When a QM forgets to start the 30 sec timer

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#72]
When a QM forgets to start the 30 sec timer, they should throw out and re-do the question immediately. The state of the quizzer answering the question should be 100% ignored.

**Comment by kclimenhaga** [comment]
I disagree. For example, if a quizmaster forgets to start the timer, but remembers after two seconds, I do not think this should invalidate the question. The 30 seconds varies in practice from room to room. Especially in a larger room, some QMs start the timer as they say the quizzer's name (or even a second before, though I think they shouldn't), while some QMs wait until the quizzer has reached the answering area/microphone. As well, I have had quizzers answer a question correctly within a second or two, occasionally before I have had a chance to start the timer. 

I agree in principle that questions should not be answered without a timer, but I think this proposed change is too extreme.

**Comment by scottpeterson** [comment]
Gryphon and I had a fun debate on the podcast regarding this. I used to think completely as you do, but then changed my mind.  

And I definitely agree with you that QMs need to be more consistent about starting the 30 sec timer. For example, the QM who doesn't start it until the quizzer reaches the mic. Someone should ask them "so, if a quizzer gives the correct answer before reaching the mic, and you heard them clearly, would you ignore that?" --because it quickly exposes how they're actually giving the quizzer > 30 sec.

**Comment by kclimenhaga** [comment]
I agree that QMs should try to be consistent; I just don't like throwing out questions when a QM makes a mistake that does not affect the quizzer's correctness. If a QM forgot to start the timer, but the quizzer answered correctly within 5 seconds, there is no question of the quizzer having been incorrect, so the QM's forgetfulness did not affect the outcome of the question. For my part, I call the quizzer's name, then start the timer. Occasionally, by the time I have cleared the 5 second countdown, the quizzer has answered, and I don't want to make the quizzer restate the answer when I know I am going to call the quizzer incorrect.

Tangentially related: would the same apply to the 5 second countdown? When I started quizmastering, I often forgot to start that countdown, because my team had old red box equipment that did not have a 5 second timer. I was absolutely counting in my head and was as consistent as possible, but should I have been throwing out every question where I forgot to start that timer?

**Comment by scottpeterson** [comment]
To your second point, yup, good catch. Anything that we decide to do regarding the 30 sec timer should apply to the 5 sec one as well. 

The crux of what Gryphon and I concluded is that it's VERY problematic to rely on the QM to know EXACTLY how many seconds have elapsed. The hypotheticals are easy to rule on if the quizzer answers correctly quickly. 

BUT, let's say the QM let 4 sec elapse. But they thought they let 2 sec elapse. So they give the quizzer 28 more sec. Quizzer answers correctly at sec 29 (REALLY sec 31). This QM has now unfairly given the quizzer more than 30 sec.

**Comment by kclimenhaga** [comment]
I guess the crux of the matter is "do we care if some quizzers get 31 seconds to answer instead of 30?" I would say no.

**Comment by scottpeterson** [comment]
That's only the crux if we think the max incorrect time elapsed judgment by a QM is plus/minus 1 sec. ;-)

**Comment by gryphonshafer** [comment]
If one quizzer gets 31 seconds and another gets 29, that's unfair. If there's an easy way to ensure every quizzer gets 30 seconds, why would we not implement it? Worst case, it's a mere inconvenience and annoyance the rare times a QM forgets to start the timer properly.

**Comment by scottpeterson** [comment]
I totally totally get that. And if QMs are forgetting to start timers a few times a quiz then this solution becomes untenable. 

I think all of this is based on a big belief of mine: things LOOK real sketchy when a QMs rulings change dependent on the outcome of what the quizzer did. 

Sure, in the scenario where the quizzer answers correctly quickly. then it doesn't matter if the QM forgot to start the timer for 2 sec or 6. 

BUT, to actually implement this, you have to wait for the quizzer to be done answering, and then make that internal calculation (without disclosing any of it to the participants). And then throw out the question in scenarios where the QM "isn't sure" if they're delay AND accurate assessment of the delay had a material impact on the outcome? it starts to optically look very bad.

**Comment by gryphonshafer** [comment]
If QMs are forgetting to start timers a few times a quiz, that seems sloppy to me. The solution then is to encourage the QMs to get better, not hide the fact that the QMs are sloppy.

I'd love to see us persistently encourage QMs to always be improving their quality. The more transparent we make everything, the better QMs will become over time.

I like to imagine increasing the transparency of everything (other than table ruling discussions), which would increase interest from spectators. If the timer is made public, so everyone in a quiz room can see it, we'd want QMs to be precise in starting it fairly for all quizzers every time.

**Comment by jswingle** [comment]
@scottpeterson is there a rule already in the rulebook which talks about this, or are you proposing we add a new rule specifically about it? I'm unsure what the action item is, here.

My take is that this doesn't need to be specified in the rulebook. No matter how solid our officials, procedural errors will occur. When that happens, the officials should consider the spirit and letter of the rulebook and make their best choice, subject to protest by the coaches of course. 

From personal experience, if I forget to start a timer but remember quickly after the quizzer is called on, I would estimate the time elapsed - let's say 3 seconds - and start the timer but call "Time" when it hits 3 seconds remaining. My estimate is probably  slightly lenient to the quizzer, but I think that's fine. We're talking a difference of perhaps 1 second, here, and with a good QM it maybe only happens a couple times a meet. If I forget the timer for a while, and truly have no clue how much time has elapsed, I would throw the question out and re-do.

I definitely think officials should not be throwing out questions "based on outcome" if by "outcome" we mean _whether the quizzer is right or wrong._ We don't want officials to be biased in favor of leniency OR in favor of strictness. BUT, there are circumstances where it is exceedingly obvious that the outcome (whether right or wrong) was not affected by the procedural error, and we should definitely not be throwing out questions in those circumstances. My rationale is that throwing out questions, though sometimes necessary, does impact the proceedings of a quiz in a large, and somewhat unfair, way. (e.g. the replaced question is unique much quicker, or the replaced F2V is much longer, etc.) It's a strong net negative that should only be resorted to when absolutely necessary to ensure fairness.

As an example: on a long FTV, the QM forgot to start the timer, and isn't sure exactly how much time elapsed, but is confident he's within 5 seconds or so of the 30 second limit. When the timer runs out, the quizzer hasn't even begun to finish the verse. It's logistically impossible for them to quote the whole verse in 5 seconds, and therefore the outcome was unaffected by the procedural error. In this case, we should absolutely NOT be throwing the question out.

**Comment by scottpeterson** [comment]
My proposal is because the QM cannot accurately know how long they forgot to start the timer for, every time they forget to start the timer they should immediately throw the question out and re-do it.

**Comment by jswingle** [comment]
But specifically, are you seeking to add this as a rule, or just a recommended practice? "They should" seems to indicate it would go in a best practices document, not the rules proper.

**Comment by scottpeterson** [comment]
I'm fairly indifferent to either of those. The sentiment seems to be it's overkill for becoming a rule. 

Though, this is making me want to go back to being a quizzer, and bring a stopwatch with me onto stage. ;-)

**Comment by gryphonshafer** [comment]
Honestly, I think this needs to be required, not just recommended. If it's just a suggestion, it'll be ignored by the very people who need to not ignore it. If a QM forgets to start the timer on 1 question every 3 quizzes, the odds of a situation showing up where it matters is tiny. But the QM that forgets to start the timer consistently on 16 of 20 questions, well, there's going to be a situation where it matters nearly every quiz.

Lest you think I'm exaggerating, I've seen otherwise good QMs start the timers as an afterthought as their routine. Every quizzer got >30 seconds in their room. Some 32 seconds; some 38 seconds. Radically unfair.

**Comment by jswingle** [comment]
> Lest you think I'm exaggerating, I've seen otherwise good QMs start the timers as an afterthought as their routine. Every quizzer got >30 seconds in their room. Some 32 seconds; some 38 seconds. Radically unfair.

I think this is the kind of egregious error better solved by official training. No amount of rulebook language is going to fix an obviously lax and unfair practice like this.

**Comment by jttower** [comment]
Just a thought... if there is a question as to whether the question was answered within 30 seconds, can't the QM/AJ just go through the recording and start the timer after they hear the name called in the recording?

**Comment by JoshJetto** [comment]
Jessica's solution is a good one in the event of a challenge based on time given to a quizzer to answer. As far as challenges based on time go, I think we need to recognize two things: 1) the quizmasters are not maliciously giving more time to some quizzers and less to others; they are working to be as fair as possible and 2) if we need a "rule" on this, there should be an allowable margin for human error based on time (like up to 2 seconds over the 30 from the time the quizzer's name is called). However, I think that the main solution written below would solve the issue. 

Some of the newer quizboxes actually have a timer that automatically starts at 32 seconds from the time the quizzer jumps. This allows 2 seconds for the quizzer to be recognized by the quizmaster, before their "official" 30 seconds starts. Perhaps another and better solution (since not all boxes have this feature) would be to have the scorekeeper have a stopwatch and start the stopwatch when the quizzer has been called by name. This would provide a failsafe in the event that the quizmaster forgot to set the timer. He/she could then call time based on the stopwatch rather than the box. I don't think this needs to be a rule in our rulebook, but could be a part of the best practices document. Also, it is unfair and disruptive to the quizzers to throw out questions just because the quizmaster forgot to start the timer, especially when it can be reasonably ascertained that they answered the question correctly within 30 seconds.  

In terms of the need for a rule concerning this, I have never seen a challenge based on the timer being started late...ever, so in some ways I feel like this is a rule looking for a problem rather than a solution to a prevailing and widespread issue of great concern to the wider, non-rules nerd, quizzing community - though I recognize the potential for unfairness and think we should mitigate that, and we may be able to do that without a rule, per se, by adding the above failsafe practices to the "best practices" document and utilizing that practice at IBQ. 

In order to have a rule on this, I think it would have to be written something like this, "The team captain or co-captain may challenge if they believe a quizzer was given an amount of time different than the 30 seconds allowed, only under the following conditions: 1) if the quizzer was called incorrect before 30 seconds had expired, and they were not out-of-context and had not given incorrect information;  2) if the quizzer gave the correct answer after 30 seconds had expired and they were called correct by the quizmaster. In order for a challenge to be upheld on the basis of too little time allowed, the amount of time between the quizzer being recognized and time being called must be less than 30 seconds _and_ it must also be reasonable to conclude that it was possible for the quizzer to have provided the correct answer had they been given a full 30 seconds. In order for a challenge to be upheld on the basis of too much time given, the amount of time between the quizzer being recognized and the correct answer being given must be greater than 32 seconds (as an allowable margin for human error).

**Comment by JoshJetto** [comment]
If we did add a rule on this, then we probably also need to add that challenges on this basis can only be made where quizzing officials are using audio recording devices to record the quizzes, so that it is possible to fairly and accurately determine the exact time that a quizzer was given to provide their answer. This, of course, is the case at Internationals, but it also provides an out for Districts that don't use audio recording from having challenges occur on the basis of time (or encourages them to use audio recording, without 'forcing' this requirement on them).

**Comment by jswingle** [comment]
I guess the major thing that confuses me about some of the discussion thus far is that quizzers/coaches can already challenge/protest if the timer is not being fairly managed by the officials. It's already in the rules that quizzers get 30 seconds... am I missing something?

> Lest you think I'm exaggerating, I've seen otherwise good QMs start the timers as an afterthought as their routine. Every quizzer got >30 seconds in their room. Some 32 seconds; some 38 seconds. Radically unfair.

This could absolutely be protested under current rules, and should be. And public display of the timer whenever possible, and certainly at Internationals, should be considered best practice so that coaches can scrutinize the actions of the officials.

That's not what the discussion was originally about, though... it's about whether we should add a rule that makes forgetting the timer require throwing the question out, always, even if the timing difference was a rounding error at best or the quizzer immediately gets the answer correct. This seems, to me, like bringing a gun to a knife fight - throwing out questions is really bad for a quiz and should only be employed when it's obvious that it was a necessary. If the timer was mistimed and a quizzer finishes their answer RIGHT on the buzzer, THEN that would be a challengeable/protestable circumstance.

**Comment by gryphonshafer** [comment]
> If the timer was mistimed and a quizzer finishes their answer RIGHT on the buzzer, THEN that would be a challengeable/protestable circumstance.

Imagine you're QMing and you forget to start the timer for 2 seconds, and so you're fully aware the timer start was delayed by a couple seconds. The quizzer completes the correct answer a half-second before the buzzer. The room cheers, but you know for a fact that they didn't complete their answer within 30 seconds. Do you count them correct (knowing they shouldn't be) and hope there's a challenge/protest? Do you count them incorrect based on your judgement of the timer delta?

**Comment by jswingle** [comment]
I would count them incorrect if they answered past whichever "judgment of the timer delta" I had decided on earlier, and just be honest with the quizzer that the timer was mistimed, the question needs to be thrown out, and it took me a while to make the full decision so in the meantime I decided to let them start answering anyways. No issue there.

**Comment by gryphonshafer** [comment]
How would you rule if the timer sound caused the quizzer to stop answering, meaning they waste the delta because they think there's no delta?

**Comment by jswingle** [comment]
> How would you rule if the timer sound caused the quizzer to stop answering, meaning they waste the delta because they think there's no delta?

Why would there be an early timer sound from a delayed timer start?

**Comment by jswingle** [comment]
@scottpeterson @gryphonshafer I think we've probably had most of the discussion necessary on this issue. Would either of you (or anyone else) like to take a stab at writing the language of a specific rule or best practice about it? I'm not passionate about it so I'm not personally going to take the time, I would just vote to close the issue unless someone else comes up with a good wording.

**Comment by scottpeterson** [comment]
I mean at this point I think all these things are true:
* No one seems to care about a couple seconds plus/minus the way Gryphon and I do.
* This is maybe best suited to be a best practice?
* If coaches/quizzers care enough, they can use a stopwatch I guess? 

I think the overall principle is still very true: it is very dangerous to wait until the conclusion of 30 seconds, or until the ruling has been made, to make the call on throwing a question out or not.

**Comment by gryphonshafer** [comment]
If I were to write something, I'd write:

> If the official responsible for tracking a quizzer's answer duration period does not start a timer when the quizzer is called, that question should be thrown out and re-asked.

But I realize everybody except Scott and I thinks this is a net-bad.

**Comment by jswingle** [comment]
^I should clarify that one of the reasons I like Gryphon's wording is that it maintains a level of of ambiguity about "when the quizzer is called" - that can be as the name of the quizzer is being said, or could be right afterward. There's still a 2 second range or so.

I know you love your objective rules, Gryphon - this rule as worded is fairly objective but also allows for the wiggle room of reality without spurious nitpicky protests.

**Comment by jttower** [comment]
Gryphon's wording is fine, but I would still prefer it being a best practice instead of a rule.
